src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static removeFromAlive(animal) {
+    const index = Animal.alive.indexOf(animal);
+
+    if (index !== -1) {
+      Animal.alive.splice(index, 1);
+    }
+  }
```

Comment:
Consider using the `filter` method on the `Animal.alive` array to remove animals with health less than or equal to zero. This approach is preferred according to the checklist and can simplify the code by avoiding the need to check for the index and use `splice`.